core/src/test/java/com/novoda/accessibility/ClosedCaptionManagerTest.java

```diff
@@ -0,0 +1,34 @@
+package com.novoda.accessibility;
+
+import android.annotation.TargetApi;
+import android.os.Build;
+import android.view.accessibility.CaptioningManager;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+@TargetApi(Build.VERSION_CODES.KITKAT)
+public class ClosedCaptionManagerTest {
+
+    @Mock
+    CaptioningManager mockCaptioningManager;
+
+    private ClosedCaptionManager closedCaptionManager;
+
+    @Before
+    public void setUp() {
+        initMocks(this);
+        closedCaptionManager = new ClosedCaptionManager(mockCaptioningManager);
+    }
+
+    @Test
+    public void whenCheckingIfCloseCaptioningEnabled_thenCaptioningManagerIsCalled() {
```

Comment:
This is different from the previous test where you checked the value returned.

https://github.com/novoda/accessibilitools/pull/2/files

In my mind, both types are valuable - the other type more so because it covers this test implicitly (if you do both cases)